Acceptance Data Package

KenwoodFox opened the pull request:
Here we go!

Docs here
https://eg-207-ccems.readthedocs.io/en/adp/

KenwoodFox committed:
Squish it all together

KenwoodFox committed:
Slightly less mess

KenwoodFox committed:
Add in interfacing information

KenwoodFox committed:
Clean up table

KenwoodFox committed:
Better wording

KenwoodFox committed:
Create space for flags

KenwoodFox committed:
Create serialEvent() untested

KenwoodFox committed:
Lowercase ACK

KenwoodFox committed:
include LC

KenwoodFox committed:
shorten example

KenwoodFox committed:
Dont need verbose wording

KenwoodFox committed:
Confirm functionality on hw

KenwoodFox committed:
Update E bytecode

KenwoodFox committed:
Make t function

KenwoodFox committed:
Implement error and warning clear + readout

KenwoodFox committed:
Fix reqs

KenwoodFox committed:
Fix command LED

KenwoodFox committed:
Fix glitches in rainflow sensor

KenwoodFox committed:
Damn, thats a lot of fun --arthur

KenwoodFox committed:
This checks out

KenwoodFox committed:
Messy min pos

KenwoodFox committed:
Upload extra forgotten commands

KenwoodFox committed:
Cleanup, rename AND add door_ajar_error as error 15

KenwoodFox committed:
Cleanup after error 15

KenwoodFox committed:
Shave bits

KenwoodFox committed:
Fix warning/error code

KenwoodFox committed:
Implement humidity instruction

MariaTswizz commented:
- [ ] 1. Inventor model files (*.ipt,*.iam) for the sensor package assembly,
- [ ] 2. Inventor STL file of the sensor housing cover and an outline drawing of the cover,
- [ ] 3. Sensor package baseplate part files and fabrication drawing,
- [ ] 4. Sensor limits and decision-making flowchart,
- [ ] 5. Characterization Plots for Visible Light and Water Collection Sensors,
- [ ] 6. System Electrical Circuit Diagram (including Arduino) with photographs of breadboard 
- [ ] setup (with cover removed),
- [ ] 7. Final LabVIEW VI file (*.vi) and Arduino Code Listing (*.ino),
- [ ] 8. Example System Data Log file, and
- [ ] 9. Verification Report (As-run Procedure performed by Instructor)

KenwoodFox committed:
Implement right and left bracket instructions

KenwoodFox committed:
Implement right and left bracket instructions

NoahR410 committed:
Added Climate Monitoring System LabVIEW

I was only able, with the time I had yesterday to get the five sensors wired but not the calibration and operation modes.

1st PRIORITY: WIRE CALIBRATION AND OPERATION MODES

KenwoodFox committed:
Create psudocode for flowchart

KenwoodFox committed:
Boilerplate PlanB.ui

KenwoodFox committed:
Merge branch 'adp' of github.com:KenwoodFox/EG-207-CCEMS into adp

KenwoodFox committed:
Implement defaulting coefiencints

KenwoodFox committed:
So much boilerplate

KenwoodFox committed:
More reqs and exit cleanly now works.

KenwoodFox committed:
implement logo, text scrolling, pyqt5 charts, pyplot and more

KenwoodFox committed:
Document reset all command

KenwoodFox committed:
Document writing temp values to temp sensor

KenwoodFox committed:
Fix typo

KenwoodFox committed:
Add polling

KenwoodFox committed:
Idk but this is not complete

KenwoodFox committed:
Confirm graph updating

NoahR410 committed:
Created Test Version of ADPcms LabVIEW

Tried to make it cleaner by adding an event structure for each key input for each sensor. HAVE NOT TESTED IT YET!!

KenwoodFox committed:
Merge branch 'adp' of github.com:KenwoodFox/EG-207-CCEMS into adp

NoahR410 committed:
Deleted folder for tested adp labview. Moved it to adpcms folder

KenwoodFox committed:
Implement L command for writing L coefs

KenwoodFox committed:
Finialize lux charactarization command

KenwoodFox committed:
Merge branch 'adp' of github.com:KenwoodFox/EG-207-CCEMS into adp

KenwoodFox committed:
prone to glitches but... error handling kinda works now

KenwoodFox committed:
Boilerplate limits on arduino and make some junk up in UI, idk how any of this is going to work out in the end.

KenwoodFox committed:
This is so crazy buggy omg XD

KenwoodFox committed:
Attempt a streamer

KenwoodFox committed:
Buggy af, but idk better than nothing at all

NoahR410 committed:
Added Calibration Module for all sensors in Labview

KenwoodFox committed:
Fix bug in planb exit cleanly, fix sensor limits

KenwoodFox commented:
![PXL_20211210_180958635.jpg](https://user-images.githubusercontent.com/31870999/145621829-5fa1e842-6e66-452e-8257-c0cc7faf1b88.jpg)
![PXL_20211210_180950826.jpg](https://user-images.githubusercontent.com/31870999/145621840-dffe62b8-784f-4463-bfee-8456e182c445.jpg)

KenwoodFox committed:
Add timeout control exception (needs to prolly get changed...)

KenwoodFox committed:
Cleanup

KenwoodFox committed:
Add better controls for automatic light sensor polling, new 'S' Instruction, new logging system and removal of bloat

KenwoodFox committed:
Put rain sensor util bar on main page

KenwoodFox committed:
Update build instructions

NoahR410 committed:
Deleted TestforADP vi file and replaced it with PotentailADP VI file

Concatenated strings used in Arduino to the device or interface specified by the VISA resource name.

KenwoodFox committed:
Revert noah

KenwoodFox committed:
Create flowchart for ADP

KenwoodFox committed:
Add spinbox to op panel

KenwoodFox committed:
Add dht11 bias to ui

KenwoodFox committed:
Fix number

KenwoodFox committed:
Fix live graph!

KenwoodFox committed:
Implement flashing sensor bias

KenwoodFox committed:
Fix photosensr UV/lux LCD bug

KenwoodFox committed:
Used for ADP

NoahR410 committed:
Update Potential ADP VI. Arduino Power section and concatenated strings
